general/clients/clients.md

```diff
@@ -62,6 +62,18 @@ A third party remote control for Jellyfin with support for Chromecast playback.
 
 * [Yatse homepage](https://yatse.tv/)
 
+## Jellyfin Cross-Platform
+
+#### Jellyfin React Client
+
+A cross-platform app using React
+
+**Status:** ✅ In development
+
+**Links:**
+
+* [GitHub](https://github.com/jellyfin/jellyfin-react-client)
+
```

Comment:
@chaosinnovator and @cromefire if you have the code for your clients hosted somewhere we can always add links here.

Reply:
UWP hacky web wrapper is [here] (https://github.com/chaosinnovator/jellyfin-uwp), functional. Xamarin app isn't functional yet but is [here] (https://github.com/chaosinnovator/jellyfin-xamarin). Want me to make a PR with one/both?

Reply:
https://github.com/cromefire/jellyfin-mpp

It would be good if you subdivide into sub clients:
Android: in development
Web/Desktop: in development
iOS: planned
Android TV: planned
Wear OS/Android auto: Maybe in the future

Reply:
@cromefire Done

Reply:
@theg00s3 if you want to add the three repositories linked above we don't need two extra pull requests, no worries though. Xamarin and MPP are cross-platform clients and the other is a web wrapper for XBox.

Reply:
@dkanada done